pym/bob/cmds/build.py

```diff
@@ -887,6 +909,13 @@ def commonBuildDevelop(parser, argv, bobRoot, develop):
     elif len(results) > 1:
         print("Build results are in:\n  ", "\n   ".join(results))
 
+    print("checkoutSteps: " + str(builder.getStatistic()['checkoutSteps']))
+    print("buildSteps:    " + str(builder.getStatistic()['buildSteps']))
+    print("packageSteps:  " + str(builder.getStatistic()['packageSteps']))
+    print("scms: " + str(builder.getStatistic()['scms']))
+    print(" -> overridden: " + str(builder.getStatistic()['scmOverrides']))
+    print("   -> packages: " + str(builder.getStatistic()['scmOverriddenPackages']))
```

Comment:
Hmm, this list could be rather long. Maybe there should be only a summary here (eg. 6 overrides active) and which packages were affected could be printed if the particular package is built ( CHECKOUT foo/bar/baz (2 scm overrides) ). What do you think?